.github/CODEOWNERS

```diff
@@ -11,6 +11,7 @@ python/dask_cudf/  @rapidsai/cudf-dask-codeowners
 cpp/CMakeLists.txt               @rapidsai/cudf-cmake-codeowners
 cpp/libcudf_kafka/CMakeLists.txt @rapidsai/cudf-cmake-codeowners
 **/cmake/                        @rapidsai/cudf-cmake-codeowners
+*.cmake                          @rapidsai/cudf-cmake-codeowners
```

Comment:
Instead of declaring cmake codeowner for all files with `.cmake` extension, we can also explicitly specify the missing file like `rapids_config.cmake`. I think the former provides more flexibility since we may add/delete files or moving them around.